CHANGELOG.md

```diff
@@ -46,10 +46,6 @@ The format is based on [these versioning and changelog guidelines](https://git.i
 
 - Upgraded `@shopify/polaris-tokens` to v2.1.1 ([#813](https://github.com/Shopify/polaris-react/pull/813))
 
-### Code quality
```

Comment:
We should keep code quality

Reply:
Sorry I thought this was `unreleased.md` 😅

Reply:
Not in the changelog, just in the unreleased